src/stories/temp-converter.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState, useEffect } from "react";
+
+export type TempConverterProps = {
+    tempNames: string[];
+};
+
+type Unit = "Celsius" | "Fahrenheit" | "Kelvin";
```

Comment:
To prevent redundancy, now that you have the `as const` array below, it would actually be better to make the type be inferred from that array. You can do this using `type Unit = typeof units[number];`.
Also please chose a unified naming convention for this term (either "unit" or "temperature unit", since "temp unit" is misleading).